core/shared/src/main/scala/org/http4s/Message.scala

```diff
@@ -84,21 +84,11 @@ sealed trait Message[+F[_]] extends Media[F] { self =>
     val entity = w.toEntity(b)
 
     val hsBase = headers ++ w.headers
-    val cl: Option[`Content-Length`] = entity.length match {
-      case Some(l) =>
-        `Content-Length`
-          .fromLong(l)
-          .fold(
-            _ => {
-              Message.logger
-                .warn(s"Attempt to provide a negative content length of $l")
```

Comment:
We are not logging a warning anymore.

Reply:
Should we log it?

Reply:
I removed this in https://github.com/http4s/http4s/pull/6645#discussion_r956575665 because it makes the ergonomics really bad. I don't know how useful it is.